doc/2/guides/main-concepts/3-permissions/index.md

```diff
@@ -235,16 +246,144 @@ Three different profiles can be created using that same role, each with varying
 }
 ```
 
+::: info
+Role restriction on indexes and collections can be used to **easily build multi-tenant application** were different groups of users cannot see each other data.
+:::
+
+### Rate limit
+
+<SinceBadge version="2.1.0" />
+
+The rate limit parameter controls how many API requests a user can send, **per second and per node**. Further requests made by a user that exceed the limit are rejected with a `429 Too Many Requests` error.
+
+::: info
+If no rate limit is defined, or if it is set to 0, then no limit is applied.
+If a user has several profiles with rate limits, the most permissive limit applies.
+:::
+
+::: warning
+Since unauthenticated users share the same user identifier, a rate limit set on the `anonymous` profile is applied to **all anonymous requests cumulated**, per second and per node. Except for the `auth:login` route, which is statically controlled in Kuzzle's configuration files.
```

Comment:
(suggestion/open to debate)

```suggestion
Since unauthenticated users share the same user identifier, a rate limit set on the `anonymous` profile is applied to **all anonymous requests cumulated**, per second and per node. Except for the `auth:login` route, which is handled separately to allow users to escalate their priviledges even if Kuzzle gets overloaded by anonymous requests.
```

Reply:
I'm not fond of the `escalate their privileges` part because privilege escalation could be a scary word :sweat_smile:

Reply:
:+1: 

I've update my suggestion.